apps/coscrad-frontend/src/store/slices/resources/shared/notes-for-resource/edge-connection-context.presenter.tsx

```diff
@@ -0,0 +1,71 @@
+import { EdgeConnectionContextType, IEdgeConnectionContext } from '@coscrad/api-interfaces';
+import { asFormattedMediaTimecodeString } from '@coscrad/media-player';
+import { Box, Typography, styled } from '@mui/material';
+import { TimeRangeMilliseconds } from '../../../../../components/resources/audio-item/audio-item-detail.full-view.presenter';
+import { convertMillisecondsToSeconds } from '../../../../../components/resources/utils/math';
+
+const TimeRangeContextVisual = (): JSX.Element => (
+    <Box
+        component="span"
+        sx={{
+            height: '20px',
+            width: '70px',
+            backgroundColor: '#75ecff',
+            borderRight: '1px solid #0671ff',
+            borderLeft: '1px solid #0671ff',
+            borderRadius: '5px',
+            display: 'inline-block',
+            ml: 1,
+            mr: 1,
+        }}
+    />
+);
+
+const HiddenData = styled('div')({
+    visibility: 'hidden',
+    height: 0,
+    width: 0,
+});
+
+type TimeRangeContext = {
```

Comment:
Shouldn't this come from API interfaces? Or have you already done custom mapping over the API response (other than just fetching it from Redux)?

Reply:
Wasn't able to resolve this.

Reply:
Ok. I went ahead and did this for you. This is the first time we've hit this, so it required a little setup work. I can see why it wasn't obvious what was needed here.